telegrambots-longpolling/src/main/java/org/telegram/telegrambots/longpolling/BotSession.java

```diff
@@ -117,7 +117,7 @@ private ScheduledFuture<?> createPollerTask() throws TelegramApiException {
                 }
             } catch (TelegramApiErrorResponseException e) {
                 long backOffMillis = backOff.nextBackOffMillis();
-                log.error("Error received from Telegram GetUpdates Request, retrying in {} millis...", backOffMillis, e);
+                log.error("Error received from Telegram GetUpdates Request, retrying in {} millis...", backOffMillis);
```

Comment:
If not printing the full stacktrace, it can be helpful if we print at least the response from telegram or the actual error why we are retrying